src/api/eth.rs

```diff
@@ -366,6 +366,14 @@ impl<T: Transport> Eth<T> {
     pub fn syncing(&self) -> CallFuture<SyncState, T::Out> {
         CallFuture::new(self.transport.execute("eth_syncing", vec![]))
     }
+
+    /// get proof
+    pub fn proof(&self, address: Address, keys: Vec<U256>, block: BlockNumber) -> CallFuture<Option<Proof>, T::Out> {
```

Comment:
```suggestion
    pub fn proof(&self, address: Address, keys: Vec<U256>, block: Option<BlockNumber>) -> CallFuture<Option<Proof>, T::Out> {
```

Let's make the block number optional and default to `Latest` to be more in line with other methods.